Latest Slesarev's Data, logging, organoleptic changes

Zhenya-Barannik opened the pull request.

Zhenya-Barannik committed:
Handeling vector of parents, graph is a true DAG now

Slesarew committed:
data: tap slants

Slesarew committed:
data: 2023-06-22

Slesarew committed:
data: backlog measurements

Slesarew committed:
data: tap new slant

Slesarew committed:
data: organoleptic 2023-06-18

Slesarew committed:
data: later 2023-06-22 data

Zhenya-Barannik committed:
Some basic data printing

Zhenya-Barannik committed:
Data 2023 06 20 (Slesarew's latest Data) (#21)

Slesarew committed:
docs: propagation and packaging

Slesarew committed:
docs: wording; correct volumes in propagation

Slesarew committed:
Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/organoleptic.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/organoleptic.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/packaging.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/packaging.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
docs: syntax and trailing sentence

Slesarew committed:
data: around 2023-06-26 tubes

Zhenya-Barannik committed:
Latest Slesarev's Data (#23)

* docs: propagation and packaging

* docs: wording; correct volumes in propagation

* Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/legacy/packaging-0.1.0.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/organoleptic.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/organoleptic.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/packaging.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* Update doc/packaging.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

* docs: syntax and trailing sentence

---------

Co-authored-by: Slesarev <slesarew@gmail.com>
Co-authored-by: Slesarew <33295157+Slesarew@users.noreply.github.com>
Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Zhenya-Barannik committed:
Merge pull request #25 from Alzymologist/data-2023-06-20

Data 2023 06 20 (Another Batch of Slesarev's data)

Zhenya-Barannik committed:
Merge pull request #26 from Alzymologist/as-propagation-docs

Repeat of Slesarev's Data Pull

Zhenya-Barannik committed:
Commented time and concentrations printing

Zhenya-Barannik committed:
"New" node and its edges are invisible now

Zhenya-Barannik committed:
Added medium, changed "participants" to "parent" for organoleptic

varovainen commented on doc/methods.md:
```diff
@@ -14,17 +14,19 @@ When pitching sample to larger one, it is more important to maintain sterility t
-40ml propagation vial: 1 g
\ No newline at end of file
+
+40ml propagation vial: 1 g
```
```suggestion
40 ml propagation vial: 1 g
```

varovainen commented on doc/yeast-capture.md:
```diff
@@ -14,9 +14,9 @@ What we focus on here is yeast purity after capture, no matter how captured stra
-2. Empty the container almost to the bottom, so that 10-50ml remains
-3. Flame the opening and add sterile wort into container; mix thoroughly
+1. Prepare approx. 50-200 ml of sterile approx. 1.040 pale wort.
+2. Empty the container almost to the bottom, so that 10-50ml remains.
```
```suggestion
2. Empty the container almost to the bottom, so that 10-50 ml remain.
```

varovainen commented on doc/yeast-capture.md:
```diff
@@ -14,9 +14,9 @@ What we focus on here is yeast purity after capture, no matter how captured stra
+2. Empty the container almost to the bottom, so that 10-50ml remains.
+3. Flame the opening and add sterile wort into container; mix thoroughly.
 4. Wait until fermentation is visible; yeast could be sampled at high krauzen.
 5. To sample, we again flame the opening of the bottle and pour some wort into sterile test tube. This wort could be used to streak the plate.
```
```suggestion
5. To sample, flame the opening of the bottle and pour some wort into sterile test tube. This wort could be used to streak the plate.
```

Slesarew committed:
Update doc/methods.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/yeast-capture.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Slesarew committed:
Update doc/yeast-capture.md

Co-authored-by: varovainen <99664267+varovainen@users.noreply.github.com>

Zhenya-Barannik committed:
Correct use of parent and participants, number_of_the_new_node

Update main.rs

Zhenya-Barannik committed:
Duplicates removed

2023-06-07-3.toml
2023-06-09-6.toml
2023-06-12-2.toml
were duplicates, see organoleptic folder

Zhenya-Barannik commented:
@Slesarew @varovainen I did everything I wanted for this specific pull request. Please review.

Zhenya-Barannik commented:
Btw, those files were duplicates, there were in both liquid and organoleptic folders. I deleted them in the liquid folder.
They were NOT renamed (as web interface on GitHub suggests). 
2023-06-07-3.toml
2023-06-09-6.toml
2023-06-12-2.toml
